locales/en/plugin__crontab-plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Actions": "",
```

Comment:
Should the message keys be empty like this?

Reply:
no it does not.. hmm not sure whats happening since I have `useKeysAsDefaultValue: true,` in my `i18next-parser.config.js`

Reply:
also `yarn i18n` scripts passes without any issue and creates the locale file :-/